### ISSUE TITLE
GSF contribution to Global Digital Compact

### ISSUE THREAD

**Issue by lmastalerz** [Green-Software-Foundation/policy-wg#36]
I was recently attending public consultations of [UN's Global Digital Compact](https://www.un.org/techenvoy/global-digital-compact) and was surprised that among nearly hundred issues contributed by the organisations there was only one mention of digital sustainability. I feel the field is very under-represented there and I was wondering if GSF could contribute there. I had a chat with Asim and he was suggesting that we could even try to contribute together with other organisations like SDIA, W3C. Deadline for the submission is March 31th.  Would that be something worth considering?

**Comment by mrchrisadams** [comment]
Hi @lmastalerz  - thanks for flagging this up! I think it would be great for the GSF to make a contribution, and they've updated the timeline to make convening a response feel more manageable now.

What's more, it looks like the deadline has been extended until later in April. I've posted the quote below:

> Submit!
> 
> You can consult with others and share the collective views, or you can submit your individual views ([see guidance notePDF](https://www.un.org/techenvoy/sites/www.un.org.techenvoy/files/Global-Digital-Compact_how-to-engage-guide.pdf)). [Share input](http://input.un.org/EFM/se/3995D1A472EC4637) by **30 April 2023** for consideration for the Global Digital Compact. See what others shared [here](http://www.un.org/techenvoy/global-digital-compact/submissions)!

@lmastalerz Would you be up for joining one the future calls to have a chat about it? We normally have them every two weeks, so the next one would have  been tomorrow on 20th, but neither myself nor @elisezelechowski would be around to chair it, so it's more likely that we'd have a call the following monday or the week afterwards on April 3rd.

If we didn't have any special meetings planned outside our usual two week meeting cadence, that would still allow for a chance to review any convened feedback on April 17th.

For what it's worth, I know that the [Association for Progressive Communications, the APC](https://limesurvey.apc.org/index.php/978733), are also carrying out a similar process, so hopefully that would take the hypothetical number of of responses referring to sustainability in the digital to at least three…

**Comment by lmastalerz** [comment]
Thanks @mrchrisadams, yes I also noticed a couple of days ago that the deadline has been shifted, so there's more time to work on this. I'm happy to join the call, in fact I already got the invite for the one planned for tomorrow March 20th, but if it's being cancelled I think I should be able to join next one on the 3rd of April. I have some ideas about the items which may be interesting in the context of GSF's submission, we can discuss details on the call.
